runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/fn/control/MSecMonitoringInfoToCounterUpdateTransformerTest.java

```diff
@@ -54,39 +58,33 @@ public void setUp() throws Exception {
 
   @Test
   public void testTransformReturnsNullIfSpecValidationFails() {
-    Map<String, String> counterNameMapping = new HashMap<>();
-    counterNameMapping.put("beam:counter:supported", "supportedCounter");
-
     Map<String, DataflowStepContext> stepContextMapping = new HashMap<>();
 
     MSecMonitoringInfoToCounterUpdateTransformer testObject =
```

Comment:
Can you add a TODO to rename this to ExecutionTimeMonitoringInfoToCounterUpdateTransformer

Reply:
Did the rename.